### ISSUE TITLE
NVDA+Firefox SVG "keyboard trap"

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#87]
This is an issue in reference to #26 which is related to the **"primitives and nested role=Menu/Menuitem"** branch. All tests use native SR navigation (arrow keys) and TAB.  

_Preface: A seasoned SR user could have the knowledge to navigate and explore the menu test webpage with more nuance, whether through manual mode switching, certain key shortcuts, etc. But this test is done by a sighted user with the SR’s default/automatic options and performed as if a new or beginner user is interacting with SVG elements. We would expect that all users could be able to navigate smoothly, regardless of experience levels._ 

<img width="806" alt="Screenshot of NVDA in a menu test environment. The focus indicator is highlighting Help 4 sub-menu checkbox" src="https://user-images.githubusercontent.com/98043076/180830195-4c86b05d-f900-4257-bece-daad8d4391be.png">

### Issue Summary
NVDA has a potential keyboard trap after the “Help” menu element. The user cannot continue navigating forward through content with the arrow keys. The arrow keys instead “scroll.” The user must press TAB to continue navigating down through the page. Doing so takes them to the HTML example menu buttons and does not read the text before it. (The end of the video shows me trying to continue navigating and experimenting with keys, but I cannot.) This could be due to NVDA’s automatic mode switching.

### Video of issue 
Example starts at 00:01:01
https://user-images.githubusercontent.com/98043076/180841640-2a24ee00-99f9-4c72-a934-dce00062e55b.mp4

### Steps to reproduce
Starting at the "Interactions Instructions" heading: Use NVDA down arrow key to navigate into the SVG space using TAB. After reaching the last "Help" menu option, press the down arrow key to continue navigation. 

### Expected behavior (Pass/Fail)
- *FAIL* We expect that a screen reader (SR) can navigate to each element with a role and aria-label using a keyboard (SVG, g, and all SVG primitives).
- *FAIL* We expect that the visual focus indicator moves to the correct element and has the correct bounding dimensions.
- *FAIL* We expect that the visual focus indicator will remain in view at all times and may even adjust the scroll position of the page while navigating.
- *Pass* The SVG is labelled by the chart's title and subtitle (these should be read as a single text when navigating to the root element).
- *FAIL* The title and subtitle are hidden from a screen reader because they are announced on their parent element instead. These elements should not be read out loud.
- *FAIL* The elements are grouped in pairs. The screen reader should be able to separately navigate to the group before navigating to each pair.
- *FAIL* Each primitive (rect, circle, path, text) should be accessible by a screen reader and read the number and label for that element.
- *FAIL/Pass* We expect that a SR will announce the aria-label correctly for each element, followed by its role.
- The aria label should announce the number, followed by the type of primitive (eg "7 Rect"), even though the text may be displayed visually in a different order. This is designed intentionally to ensure that the ARIA is working correctly.
- *FAIL* We expect that different SVG primitives (rect, circle, path, text) will be consistent with each other based on previous expectations.
- *FAIL* We expect that the SVG elements will provide the same experience as the HTML elements (below the SVG space).
- *Pass* We expect that a SR will be able to expand a menu element open or closed.
- *FAIL* We expect that a SR will announce the expanded state correctly when navigated to.
- *FAIL* We expect that a SR will announce the expanded state correctly when changed.
- *Pass* We expect that when an option has been selected, the SR will give feedback.

### Technical Details
NVDA 2022.1 (default)
Firefox 102.0.1
Windows 11 Build 22000.795
